### PR TITLE
New mtl releases blogpost

### PR DESCRIPTION
**Pull request by philipandag**

### CODE REVIEW

File: blog/content/post/2024-07-23-new-mtl-releases.md [hunk 1]
```diff
@@ -0,0 +1,310 @@
...
+- <https://review.coreboot.org/c/coreboot/+/82673>
+- <https://review.coreboot.org/c/coreboot/+/82686>
+- <https://review.coreboot.org/c/coreboot/+/82685>
+- <https://review.coreboot.org/c/coreboot/+/82674>
+- <https://review.coreboot.org/c/coreboot/+/82898>
+
+### Linux kernel
+
+- <https://github.com/torvalds/linux/commit/e1c6db864599be341cd3bcc041540383215ce05e>
+
```
Comment (mkopec) [inline blog/content/post/2024-07-23-new-mtl-releases.md:279]: We can add these 2 to the list:
- https://github.com/MrChromebox/edk2/commit/7f398d3b14928ffecbbe92bd93213db91dad7703
- https://github.com/systemd/systemd/commit/f5c8dd85ee13f9308498faf6a0c4837e604f8dcb

File: blog/content/post/2024-07-23-new-mtl-releases.md [hunk 2]
```diff
@@ -0,0 +1,310 @@
...
+The validation procedure is described at [laboratory assembly guide](https://docs.dasharo.com/transparent-validation/novacustom/laboratory-assembly-guide/#prerequisites).
+Details about specific test cases can be found in our [Dasharo Test Specification]\
+(<https://docs.dasharo.com/unified-test-documentation/overview/>)
+
+## Our contributions
+
+While working on the new release we have made multiple contributions
+open source projects:
+
+### Coreboot
```
Comment (mkopec) [inline blog/content/post/2024-07-23-new-mtl-releases.md:266]:
```suggestion
### coreboot
```

and all other instances if applicable

File: blog/content/post/2024-07-23-new-mtl-releases.md (hunk 1 above)
Comment (philipandag) [reply blog/content/post/2024-07-23-new-mtl-releases.md:279]: added

File: blog/content/post/2024-07-23-new-mtl-releases.md (hunk 2 above)
Comment (philipandag) [reply blog/content/post/2024-07-23-new-mtl-releases.md]: Oh, the name is lowercase, I haven't noticed that, thanks! Fixed in [c50166d](https://github.com/3mdeb/news-and-ideas/pull/598/commits/c50166d3159eb2a2e5f7b67b13ce2c46a7329a46)

File: blog/content/post/2024-07-23-new-mtl-releases.md [hunk 3]
```diff
@@ -0,0 +1,329 @@
...
+about that event will be displayed on the screen. For more details
+check the link in the header.
+
+![Vboot Verified Boot popup](/img/verified_boot_popup.png)
+
+### [UEFI Shell](https://docs.dasharo.com/unified-test-documentation/dasharo-compatibility/30P-uefi-shell/)
+
+UEFI Shell is a command line interface that allows for interacting
+with the UEFI firmware. It can be used for diagnosing, debugging,
+configuring and updating the firmware. Check the link in the header.
```
Comment (mkopec) [inline blog/content/post/2024-07-23-new-mtl-releases.md:75]:
```suggestion
and configuring the firmware. Check the link in the header.
```

File: blog/content/post/2024-07-23-new-mtl-releases.md [hunk 4]
```diff
@@ -0,0 +1,329 @@
...
+![UEFI Shell](/img/uefi_shell_v2.2.png)
+
+### [UEFI Secure Boot](https://docs.dasharo.com/unified-test-documentation/dasharo-security/206-secure-boot/)
+
+Secure Boot is a fundamental security feature of UEFI specification.
+Thanks to secure boot, the device will only boot firmware components
```
Comment (mkopec) [inline blog/content/post/2024-07-23-new-mtl-releases.md:81]:
```suggestion
Thanks to secure boot, the device will only boot operating systems
```

File: blog/content/post/2024-07-23-new-mtl-releases.md [hunk 5]
```diff
@@ -0,0 +1,329 @@
...
+which are trusted using cryptographic signatures embedded in the
+firmware. Check the link in the header for more details.
+![UEFI Secure Boot](/img/secure_boot.png)
+
+### [Automatic Embedded Controller update](https://docs.dasharo.com/unified-test-documentation/dasharo-compatibility/31G-ec-and-superio/#ecr031001-ec-firmware-sync-in-coreboot)
+
+The feature allows updating the Embedded Controller firmware easily
+alongside updating the bios firmware.
```
Comment (mkopec) [inline blog/content/post/2024-07-23-new-mtl-releases.md:89]:
```suggestion
alongside updating the BIOS firmware.
```

File: blog/content/post/2024-07-23-new-mtl-releases.md [hunk 6]
```diff
@@ -0,0 +1,329 @@
...
+System Management Mode privileged code. It prevents flashing
+the firmware with programs like [flashrom](https://www.flashrom.org/).
+![SMM BIOS write protection](/img/smm_bios_write_protection.png)
+
+### [Early boot DMA protection](https://docs.dasharo.com/dasharo-menu-docs/dasharo-system-features/#dasharo-security-options)
+
+The IOMMU DMA protection enabled early
+in the POST process prevents Direct Memory Access attacks.
+Leaking secrets and injecting malware using devices such
+as PCIe cards and USB4/thunderbolt devices is prevented
```
Comment (mkopec) [inline blog/content/post/2024-07-23-new-mtl-releases.md:123]:
```suggestion
as PCIe cards and USB4/Thunderbolt devices is prevented
```

File: blog/content/post/2024-07-23-new-mtl-releases.md [hunk 7]
```diff
@@ -0,0 +1,329 @@
...
+The Early Sign of Life display is a feature that displays the
+firmware version during the boot phase. It is useful for
+diagnosing boot issues and verifying that the firmware
+is up-to-date.
+
+### [Current limiting for USB-PD power supplies](https://docs.dasharo.com/unified-test-documentation/dasharo-compatibility/31H-usb-type-c/#utc020001-usb-type-c-pd-current-limiting-ubuntu-2204)
+
+Limiting the current draw from USB-Power Delivery is an
+important safety feature that controls the maximum current
+that can flow through the USB-PD. This prevents damaging the device
```
Comment (mkopec) [inline blog/content/post/2024-07-23-new-mtl-releases.md:137]:
```suggestion
that can flow through the USB-PD power supply. This prevents damaging the device
```

File: blog/content/post/2024-07-23-new-mtl-releases.md [hunk 8]
```diff
@@ -0,0 +1,329 @@
...
+as well as what it is connected to in case of any malfunction
+like short-circuiting the USB port.
```
Comment (mkopec) [inline blog/content/post/2024-07-23-new-mtl-releases.md:139]: Not really, this is just about respecting the limits that the USB-C charger tells us. Before, the laptop would just pull as many watts as it wanted from a charger, overdrawing from it. Most chargers would engage overcurrent protection at this point and reset or shutdown, but some cheaper chargers could be damaged if this was done repeatedly.

File: blog/content/post/2024-07-23-new-mtl-releases.md [hunk 9]
```diff
@@ -0,0 +1,329 @@
...
+### [Built-in webcam disable option in setup menu](https://docs.dasharo.com/dasharo-menu-docs/dasharo-system-features/#dasharo-security-options)
+
+With this option disabled the power to the integrated webcam
+is cut off making it as good as physically removed from the device
+until the camera is enabled again.
+![Webcam disable option](/img/enable_camera.png)
+
+### [USB stack disable option in setup menu](https://docs.dasharo.com/dasharo-menu-docs/dasharo-system-features/#usb-configuration)
+
+The option controls loading of USB firmware USB drivers. Disabling
```
Comment (mkopec) [inline blog/content/post/2024-07-23-new-mtl-releases.md:162]:
```suggestion
The option controls loading of firmware USB drivers. Disabling
```

File: blog/content/post/2024-07-23-new-mtl-releases.md (hunk 3 above)
Comment (philipandag) [reply blog/content/post/2024-07-23-new-mtl-releases.md:75]: thanks!

File: blog/content/post/2024-07-23-new-mtl-releases.md (hunk 8 above)
Comment (philipandag) [reply blog/content/post/2024-07-23-new-mtl-releases.md]: [5d83859](https://github.com/3mdeb/news-and-ideas/pull/598/commits/5d838599cca210aea0fc9571724be1a9c17bb9b1)

**Comment by miczyg1** [comment]
@philipandag please set the PR base to develop. We never prepare a blog post for master directly. https://github.com/3mdeb/news-and-ideas?tab=readme-ov-file#deployment-on-beta-blog

**Comment by philipandag** [comment]
@miczyg1 Sure, thanks for the tip. It's now rebased to `develop`.

File: blog/content/post/2024-07-23-new-mtl-releases.md [hunk 10]
```diff
@@ -0,0 +1,333 @@
...
+## Test coverage
+
+The tests were performed on NovaCustom V540TU and V560TU platforms.
+Checkout the results at our
+[osfv-results repo](https://github.com/Dasharo/osfv-results):
+
+- [V540TU](https://github.com/Dasharo/osfv-results/blob/main/boards/NovaCustom/MTL_14th_Gen/V540TU/v0.9.0-results.csv)
+- [V560TU](https://github.com/Dasharo/osfv-results/blob/main/boards/NovaCustom/MTL_14th_Gen/V560TU/v0.9.0-results.csv)
+
+The validation procedure is described at [laboratory assembly guide](https://docs.dasharo.com/transparent-validation/novacustom/laboratory-assembly-guide/#prerequisites).
```
Comment (BeataZdunczyk) [inline blog/content/post/2024-07-23-new-mtl-releases.md:263]: 404, as it should be. We decided not to use this setup anymore.

**Review by BeataZdunczyk — DISMISSED** [review]

File: blog/content/post/2024-07-23-new-mtl-releases.md [hunk 11]
```diff
@@ -0,0 +1,333 @@
...
+We are thrilled to announce that the support for newest Intel
+Meteor Lake platform has been added to Dasharo in release v0.9.0!
+The newest release features numerous new features and improvements
+related to security, functionality and quality of life.
+Thanks to everyone who contributed to this release!
+
+## Added features
+
+Here is a brief introduction to the changes in Dasharo v0.9.0,
+for more details check out:
```
[the remark this answers is not in the document]
Comment (philipandag) [reply blog/content/post/2024-07-23-new-mtl-releases.md:37]: [e278f32](https://github.com/3mdeb/news-and-ideas/pull/598/commits/e278f325514e50e83528873c1d8c1bbef13a7cb0)

File: blog/content/post/2024-07-23-new-mtl-releases.md [hunk 12]
```diff
@@ -0,0 +1,333 @@
...
+
+### [UEFI Secure Boot](https://docs.dasharo.com/unified-test-documentation/dasharo-security/206-secure-boot/)
+
+Secure Boot is a fundamental security feature of UEFI specification.
+Thanks to secure boot, the device will only boot operating systems
+which are trusted using cryptographic signatures embedded in the
+firmware. Check the link in the header for more details.
+![UEFI Secure Boot](/img/secure_boot.png)
+
+### [Automatic Embedded Controller update](https://docs.dasharo.com/unified-test-documentation/dasharo-compatibility/31G-ec-and-superio/#ecr031001-ec-firmware-sync-in-coreboot)
```
[the remark this answers is not in the document]
Comment (philipandag) [reply blog/content/post/2024-07-23-new-mtl-releases.md:86]: I was just copying the new features from https://docs.dasharo.com/variants/novacustom_v540tu/releases/#v090-2024-07-18

File: blog/content/post/2024-07-23-new-mtl-releases.md [hunk 13]
```diff
@@ -0,0 +1,333 @@
...
+### [Block boot when battery is too low](https://docs.dasharo.com/unified-test-documentation/dasharo-compatibility/359-boot-blocking/#test-cases-common-documentation)
+
+With this feature enabled, the device will not boot if the battery
+is below a certain threshold. The reason for this it to prevent
+a situation where the charge would become too low
+to support the boot process and the device would unexpectedly shut down
+which could lead to data loss or corruption.
+![Battery block popup](/img/battery_block_popup.jpg)
+
+### [Power on AC option in setup menu](https://docs.dasharo.com/dasharo-menu-docs/dasharo-system-features/#power-management-options)
```
[the remark this answers is not in the document]
Comment (philipandag) [reply blog/content/post/2024-07-23-new-mtl-releases.md:209]: To my defence, the titles here are also the same as in https://docs.dasharo.com/variants/novacustom_v540tu/releases/#v090-2024-07-18

File: blog/content/post/2024-07-23-new-mtl-releases.md (hunk 10 above)
Comment (philipandag) [reply blog/content/post/2024-07-23-new-mtl-releases.md:263]: It is, again, copied directly from the releases page. https://docs.dasharo.com/variants/novacustom_v540tu/releases/#v090-2024-07-18 Do I delete this part?

Comment (BeataZdunczyk) [reply blog/content/post/2024-07-23-new-mtl-releases.md:263]: yes, please

Comment (philipandag) [reply blog/content/post/2024-07-23-new-mtl-releases.md]: @BeataZdunczyk [1ad7b30](https://github.com/3mdeb/news-and-ideas/pull/598/commits/1ad7b30522e1139e769986e08325f1794dcaf88a)

File: blog/content/post/2024-07-23-new-mtl-releases.md (hunk 12 above)
[the remark this answers is not in the document]
Comment (BeataZdunczyk) [reply blog/content/post/2024-07-23-new-mtl-releases.md:86]: please removed this one @philipandag .
@mkopec  please check if there are any features that need to be removed and let's publish this blog post.

File: blog/content/post/2024-07-23-new-mtl-releases.md [hunk 14]
```diff
@@ -0,0 +1,333 @@
...
+### [Firmware update mode](https://docs.dasharo.com/guides/firmware-update/#firmware-update-mode)
+
+The Firmware Update Mode is a quick way of ensuring your device
+is prepared for a firmware update. It temporarily changes the
+configuration and returns it to the previous state after
+the update is completed. Check the link in the header for more details
+and instructions on how to use it.
+![Firmware Update Mode Confirmation Screen](/img/setup_menu_fum_confirmation.png)
+
+### [BIOS boot medium write-protection](https://docs.dasharo.com/dasharo-menu-docs/dasharo-system-features/#dasharo-security-options)
```
[the remark this answers is not in the document]
Comment (mkopec) [reply blog/content/post/2024-07-23-new-mtl-releases.md:101]: That's the way the option is called, we should try to stay consistent. @philipandag please fix

File: blog/content/post/2024-07-23-new-mtl-releases.md (hunk 12 above)
[the remark this answers is not in the document]
Comment (mkopec) [reply blog/content/post/2024-07-23-new-mtl-releases.md:86]: I don't see any more removed features

[the remark this answers is not in the document]
Comment (philipandag) [reply blog/content/post/2024-07-23-new-mtl-releases.md:86]: Should I also update the release notes on docs.dasharo.com?

[the remark this answers is not in the document]
Comment (philipandag) [reply blog/content/post/2024-07-23-new-mtl-releases.md]: removed: [252e077](https://github.com/3mdeb/news-and-ideas/pull/598/commits/252e07736c4ad1e934185b25d0e21ed0aadc4617)

[the remark this answers is not in the document]
Comment (philipandag) [reply blog/content/post/2024-07-23-new-mtl-releases.md]: Removed: [252e077](https://github.com/3mdeb/news-and-ideas/pull/598/commits/252e07736c4ad1e934185b25d0e21ed0aadc4617)

[the remark this answers is not in the document]
Comment (mkopec) [reply blog/content/post/2024-07-23-new-mtl-releases.md]: > Should I also update the release notes on docs.dasharo.com?

yes please :)

[the remark this answers is not in the document]
Comment (philipandag) [reply blog/content/post/2024-07-23-new-mtl-releases.md]: PR in docs https://github.com/Dasharo/docs/pull/877

**Comment by mkopec** [comment]
@philipandag Please try to squash some of the commits before we merge this

**Comment by philipandag** [comment]
squashed a bit

**Review by mkopec — APPROVED** [review]
Looks good now

File: blog/content/post/2024-07-23-new-mtl-releases.md [hunk 15]
```diff
@@ -0,0 +1,333 @@
...
+device tried to draw too much current, but triggering
+it repeatedly could shorten the lifespan of the charger
+or simply damage it.
+
+### [Setup menu password configuration](https://docs.dasharo.com/dasharo-menu-docs/overview/$user-password-management)
+
+Setting up a password for the setup menu allows for
+protecting the configuration from unauthorized changes.
+
+### [Wi-Fi / Bluetooth module disable option in setup menu](https://docs.dasharo.com/dasharo-menu-docs/dasharo-system-features/#dasharo-security-options)
```
[the remark this answers is not in the document]
Comment (mkopec) [reply blog/content/post/2024-07-23-new-mtl-releases.md]: fixed in d90da1784289f180135eb1bb38dd84fbaf5f36ff

File: blog/content/post/2024-07-23-new-mtl-releases.md [hunk 16]
```diff
@@ -0,0 +1,333 @@
...
+
+With this option enabled, the Wi-Fi and Bluetooth modules
+are powered off and disabled. Use this when you want to
+ensure that the device is not transmitting any data wirelessly.
+![Wi-Fi / Bluetooth module disable option](/img/enable_wifi_bt.png)
+
+### [Built-in webcam disable option in setup menu](https://docs.dasharo.com/dasharo-menu-docs/dasharo-system-features/#dasharo-security-options)
```
[the remark this answers is not in the document]
Comment (mkopec) [reply blog/content/post/2024-07-23-new-mtl-releases.md]: fixed in d90da1784289f180135eb1bb38dd84fbaf5f36ff

File: blog/content/post/2024-07-23-new-mtl-releases.md [hunk 17]
```diff
@@ -0,0 +1,333 @@
...
+
+With this option disabled the power to the integrated webcam
+is cut off making it as good as physically removed from the device
+until the camera is enabled again.
+![Webcam disable option](/img/enable_camera.png)
+
+### [USB stack disable option in setup menu](https://docs.dasharo.com/dasharo-menu-docs/dasharo-system-features/#usb-configuration)
```
[the remark this answers is not in the document]
Comment (mkopec) [reply blog/content/post/2024-07-23-new-mtl-releases.md]: fixed in d90da1784289f180135eb1bb38dd84fbaf5f36ff

File: blog/content/post/2024-07-23-new-mtl-releases.md [hunk 18]
```diff
@@ -0,0 +1,333 @@
...
+
+The option controls loading of firmware USB drivers. Disabling
+it will prevent the USB devices from working until an OS
+with a proper USB driver is loaded.
+![Enable USB stack](/img/enable_usb_stack.png)
+
+### [Network stack disable option in setup menu](https://docs.dasharo.com/dasharo-menu-docs/dasharo-system-features/#networking-options)
```
[the remark this answers is not in the document]
Comment (mkopec) [reply blog/content/post/2024-07-23-new-mtl-releases.md]: fixed in d90da1784289f180135eb1bb38dd84fbaf5f36ff

File: blog/content/post/2024-07-23-new-mtl-releases.md (hunk 14 above)
[the remark this answers is not in the document]
Comment (mkopec) [reply blog/content/post/2024-07-23-new-mtl-releases.md]: fixed in d90da1784289f180135eb1bb38dd84fbaf5f36ff

File: blog/content/post/2024-07-23-new-mtl-releases.md [hunk 19]
```diff
@@ -0,0 +1,333 @@
...
+![BIOS boot medium write-protection](/img/bios_lock.png)
+
+### [SMM BIOS write protection](https://docs.dasharo.com/dasharo-menu-docs/dasharo-system-features/#dasharo-security-options)
+
+With this option enabled, the BIOS can only be modified by
+System Management Mode privileged code. It prevents flashing
+the firmware with programs like [flashrom](https://www.flashrom.org/).
+![SMM BIOS write protection](/img/smm_bios_write_protection.png)
+
+### [Early boot DMA protection](https://docs.dasharo.com/dasharo-menu-docs/dasharo-system-features/#dasharo-security-options)
```
[the remark this answers is not in the document]
Comment (mkopec) [reply blog/content/post/2024-07-23-new-mtl-releases.md]: fixed in d90da1784289f180135eb1bb38dd84fbaf5f36ff

File: blog/content/post/2024-07-23-new-mtl-releases.md [hunk 20]
```diff
@@ -0,0 +1,333 @@
...
+firmware. Check the link in the header for more details.
+![UEFI Secure Boot](/img/secure_boot.png)
+
+### [Automatic Embedded Controller update](https://docs.dasharo.com/unified-test-documentation/dasharo-compatibility/31G-ec-and-superio/#ecr031001-ec-firmware-sync-in-coreboot)
+
+The feature allows updating the Embedded Controller firmware easily
+alongside updating the BIOS firmware.
+Check out the link in the section title for more details.
+
+### [Firmware update mode](https://docs.dasharo.com/guides/firmware-update/#firmware-update-mode)
```
[the remark this answers is not in the document]
Comment (mkopec) [reply blog/content/post/2024-07-23-new-mtl-releases.md]: fixed in d90da1784289f180135eb1bb38dd84fbaf5f36ff

**Pull request merged**